develop-image/Dockerfile

```diff
@@ -44,25 +43,22 @@ RUN apt-get install -y                                                       \
 
 
 # needed by Pipenv
-ENV DEBIAN_FRONTEND noninteractive
 ENV LC_ALL C.UTF-8
 ENV LANG C.UTF-8
 
-RUN apt-get install -y tox
-RUN python3.7 -m pip install -U pipenv
-
-ENV PATH="/usr/local/bin:${PATH}"
-USER default
```

Comment:
Not needed?

Reply:
I dont think so, 
everything seems to work in the image as I would expected.